File: drone/position.py
class position:

    def __init__(self, latitude, longitude, altitude):
        self.latitude = latitude
        self.longitude = longitude
        self.altitude = altitude

    def setLatitude(self, latitude):
        self.latitude = latitude

    def getLatitude(self):
        return self.latitude

    def setLongitude(self, longitude):
        self.longitude = longitude

    def getLongitude(self):
        return self.longitude

    def setAltitude(self, altitude):
        self.altitude = altitude

    def getAltitude(self):
        return self.altitude

File: android/app/src/main/java/ila/fr/codisintervention/models/model/Position.java
package ila.fr.codisintervention.models.model;

import ila.fr.codisintervention.models.messages.Location;
import lombok.Getter;
import lombok.Setter;

/**
 * Represents coordinates in terms of latitude, longitude and altitude.
 */
@Getter
@Setter
public class Position {

    /** The latitude of the position */
    private double latitude;

    /** The longitude of the position */
    private double longitude;


    public Position(double latitude, double longitude) {
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public Position(Location loc) {
        latitude = loc.getLat();
        longitude = loc.getLng();
    }

    public void load(Position pos){
        latitude = pos.getLatitude();
        longitude = pos.getLongitude();
    }
}
